styles/abstracts/_mixins/transitions.scss

```diff
@@ -0,0 +1,10 @@
+@mixin transition($property) {
+  -webkit-transition: $property;
+  -moz-transition: $property;
+  -o-transition: $property;
+  transition: $property;
```

Comment:
PostCSS should handle the browser compatibility

Reply:
Gucci